old-datastore/reference/data-api.rst

```diff
@@ -45,7 +45,7 @@ Example API call:
 reporting-org
 `````````````
 
-Returns activities where the `reporting-org <http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/>`__ element has a @ref attribute value that matches your specified `Organisation identifier <http://iatistandard.org/organisation-identifiers/>`__ string.
+Returns activities where the `reporting-org <http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/>`__ element has a @ref attribute value that matches your specified `Organisation identifier <https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/>`__ string.
```

Comment:
The link on “Organisation identifier” here points to [reporting-org](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/). Is that intended?

Reply:
Yes

Reply:
So the sentence will read like this:

> #### reporting-org
> Returns activities where the [reporting-org](http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/) element has a `@ref` attribute value that matches your specified [Organisation identifier](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/) string.

…where both links point to the same page.

Intuitively, I’d expect these links to go to different places.

Reply:
Agreed. If you would like, please do re-write/update the documentation and send in a PR.

At the moment I'm just fixing links and aligning them to the new content/structure. From this point on we're not going to be improving the guidance for the old datastore. But happy for you to do so.

Reply:
I can’t tell if we’re speaking at cross-purposes.

The link previously pointed here:
http://iatistandard.org/organisation-identifiers/

I think it should have been changed to point here:
http://iatistandard.org/203/organisation-identifiers/

Instead, this PR changed it to point here:
https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/

That’s not the same intended location, so I don’t think it fixes the link.